### PR TITLE
GP02HYPM deploy csv updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updates to GP02HYPM deploy csv per metadata review
Also removed unnecessary notes
Updated CUIDs per recent CruiseInformation.csv updates to align with R2R CUID formats

Refer to attached spreadsheet for HYPM sensor depths.

[Subsurface_Mooring_Depth_Details_2020-05-29.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/4712444/Subsurface_Mooring_Depth_Details_2020-05-29.xlsx)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Critical Metadata Revew [DT-15]](https://trello.com/c/OuquTzsl/15-critical-metadata-revew-dt-15)

File: deployment/GP02HYPM_Deploy.csv [hunk 1]
```diff
@@ -1,99 +1,99 @@
...
+TN323,,RB1605,,GP02HYPM-WFP03-00-WFPENG000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGCON-GWFPGX-10403,50.07983,-144.803,,4070,4219,
+TN323,,RB1605,,GP02HYPM-WFP03-01-FLORDL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-FLORDL-03284,50.07983,-144.803,,4070,4219,
+TN323,,RB1605,,GP02HYPM-WFP03-03-DOSTAL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-DOSTAL-01478,50.07983,-144.803,,4070,4219,
+TN323,,RB1605,,GP02HYPM-WFP03-04-CTDPFL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-CTDPFL-00132,50.07983,-144.803,,4070,4219,
+TN323,,RB1605,,GP02HYPM-WFP03-05-VEL3DL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,50.07983,-144.803,,4070,4219,
+#TN323,,RB1605,,GP02HYPM-MPM01-02-ZPLSGA009,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,,,50.07983,-144.803,,,4220,Not Deployed
+#TN323,,RB1605,,GP02HYPM-MPM01-02-ZPLSGA010,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,,,50.07983,-144.803,,,4220,Not Deployed
```
Comment (snwhite) [inline deployment/GP02HYPM_Deploy.csv:43]: Deployment 3: ZPLSG deployment depths were correct at 4219 (should not have been changed).  
Single recovery time is correct – mid-water release failed, bottom release valid for all components.

File: deployment/GP02HYPM_Deploy.csv [hunk 2]
```diff
@@ -1,99 +1,99 @@
...
+RB1605,,SR1710,,GP02HYPM-WFP02-05-VEL3DL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-10402,CGINS-VEL3DL-01108,50.0796,-144.80572,,2094,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-00-WFPENG000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGCON-GWFPGX-93602,50.0796,-144.80572,,4062,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-01-FLORDL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-FLORDL-02736,50.0796,-144.80572,,4062,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-03-DOSTAL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-DOSTAL-01086,50.0796,-144.80572,,4062,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-04-CTDPFL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-CTDPFL-00107,50.0796,-144.80572,,4062,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-05-VEL3DL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-VEL3DL-01024,50.0796,-144.80572,,4062,4220,
```
Comment (snwhite) [inline deployment/GP02HYPM_Deploy.csv:57]: Deployment 4: WFP03 and SIOENG stopDateTime should be 2017-07-16T18:24 (per mooring recovery log).

File: deployment/GP02HYPM_Deploy.csv [hunk 3]
```diff
@@ -1,99 +1,99 @@
...
+SR1710,,SR1811,,GP02HYPM-WFP02-05-VEL3DL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-99701,CGINS-VEL3DL-01026,50.1156,-144.9176,,2095,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-00-WFPENG000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGCON-GWFPGX-10403,50.1156,-144.9176,,4063,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-01-FLORDL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGINS-FLORDL-03284,50.1156,-144.9176,,4063,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-03-DOSTAL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGINS-DOSTAL-01478,50.1156,-144.9176,,4063,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-04-CTDPFL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGINS-CTDPFL-00132,50.1156,-144.9176,,4063,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-05-VEL3DL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,50.1156,-144.9176,,4063,4230,
```
Comment (snwhite) [inline deployment/GP02HYPM_Deploy.csv:71]: Deployment 5 looks good.
Single recovery time is correct – mid-water release failed, bottom release valid for all components.

File: deployment/GP02HYPM_Deploy.csv [hunk 4]
```diff
@@ -1,99 +1,99 @@
...
+SR1811,,SKQ201920S,,GP02HYPM-WFP02-05-VEL3DL000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-10402,CGINS-VEL3DL-01108,50.06856,-144.79826,,2095,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-00-WFPENG000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGCON-GWFPGX-93602,50.06856,-144.79826,,4063,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-01-FLORDL000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGINS-FLORDL-02736,50.06856,-144.79826,,4063,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-03-DOSTAL000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGINS-DOSTAL-01086,50.06856,-144.79826,,4063,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-04-CTDPFL000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGINS-CTDPFL-00107,50.06856,-144.79826,,4063,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-05-VEL3DL000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGINS-VEL3DL-01024,50.06856,-144.79826,,4063,4222,
```
Comment (snwhite) [inline deployment/GP02HYPM_Deploy.csv:85]: Deployment 6: WFP03 and SIOENG stopDateTime should be 2019-09-26T22:09 (per mooring recovery log).

File: deployment/GP02HYPM_Deploy.csv [hunk 5]
```diff
@@ -1,99 +1,99 @@
...
+SKQ201920S,,,,GP02HYPM-WFP02-04-CTDPFL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-99701,CGINS-CTDPFL-00115,50.11518,-144.91612,,2095,4237,
+SKQ201920S,,,,GP02HYPM-WFP02-05-VEL3DL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-99701,CGINS-VEL3DL-01026,50.11518,-144.91612,,2095,4237,
+SKQ201920S,,,,GP02HYPM-WFP03-00-WFPENG000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGCON-GWFPGX-10403,50.11518,-144.91612,,4063,4237,
+SKQ201920S,,,,GP02HYPM-WFP03-01-FLORDL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-FLORDL-03284,50.11518,-144.91612,,4063,4237,
+SKQ201920S,,,,GP02HYPM-WFP03-03-DOSTAL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-DOSTAL-01478,50.11518,-144.91612,,4063,4237,
+SKQ201920S,,,,GP02HYPM-WFP03-04-CTDPFL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-CTDPFL-00132,50.11518,-144.91612,,4063,4237,
+SKQ201920S,,,,GP02HYPM-WFP03-05-VEL3DL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,50.11518,-144.91612,,4063,4237,
```
Comment (snwhite) [inline deployment/GP02HYPM_Deploy.csv:99]: Deployment 7 looks good.

File: deployment/GP02HYPM_Deploy.csv (hunk 1 above)
Comment (rtravis-whoi) [reply deployment/GP02HYPM_Deploy.csv:43]: oh whoops - looks like a copy/paste error fr total depth on next deployment - fixed

File: deployment/GP02HYPM_Deploy.csv (hunk 2 above)
Comment (rtravis-whoi) [reply deployment/GP02HYPM_Deploy.csv:57]: done

File: deployment/GP02HYPM_Deploy.csv (hunk 4 above)
Comment (rtravis-whoi) [reply deployment/GP02HYPM_Deploy.csv:85]: done

File: deployment/GP02HYPM_Deploy.csv [hunk 6]
```diff
@@ -1,99 +1,99 @@
...
+TN323,,RB1605,,GP02HYPM-WFP03-00-WFPENG000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGCON-GWFPGX-10403,50.07983,-144.803,,4070,4219,
+TN323,,RB1605,,GP02HYPM-WFP03-01-FLORDL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-FLORDL-03284,50.07983,-144.803,,4070,4219,
+TN323,,RB1605,,GP02HYPM-WFP03-03-DOSTAL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-DOSTAL-01478,50.07983,-144.803,,4070,4219,
+TN323,,RB1605,,GP02HYPM-WFP03-04-CTDPFL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-CTDPFL-00132,50.07983,-144.803,,4070,4219,
+TN323,,RB1605,,GP02HYPM-WFP03-05-VEL3DL000,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,50.07983,-144.803,,4070,4219,
+#TN323,,RB1605,,GP02HYPM-MPM01-02-ZPLSGA009,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,,,50.07983,-144.803,,,4219,Not Deployed
+#TN323,,RB1605,,GP02HYPM-MPM01-02-ZPLSGA010,3,1,2015-06-04T23:38:00,2016-07-01T17:57:00,CGMGP-02HYPM-00003,,,50.07983,-144.803,,,4219,Not Deployed
```
Comment (snwhite) [inline deployment/GP02HYPM_Deploy.csv:43]: Deployment 3: Correction looks good.
Single recovery time is correct – mid-water release failed, bottom release valid for all components.

File: deployment/GP02HYPM_Deploy.csv [hunk 7]
```diff
@@ -1,99 +1,99 @@
...
+RB1605,,SR1710,,GP02HYPM-WFP02-05-VEL3DL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-10402,CGINS-VEL3DL-01108,50.0796,-144.80572,,2094,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-00-WFPENG000,4,1,2016-07-03T02:05:00,2017-07-16T18:24:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGCON-GWFPGX-93602,50.0796,-144.80572,,4062,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-01-FLORDL000,4,1,2016-07-03T02:05:00,2017-07-16T18:24:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-FLORDL-02736,50.0796,-144.80572,,4062,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-03-DOSTAL000,4,1,2016-07-03T02:05:00,2017-07-16T18:24:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-DOSTAL-01086,50.0796,-144.80572,,4062,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-04-CTDPFL000,4,1,2016-07-03T02:05:00,2017-07-16T18:24:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-CTDPFL-00107,50.0796,-144.80572,,4062,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-05-VEL3DL000,4,1,2016-07-03T02:05:00,2017-07-16T18:24:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-VEL3DL-01024,50.0796,-144.80572,,4062,4220,
```
Comment (snwhite) [inline deployment/GP02HYPM_Deploy.csv:57]: Deployment 4: Corrections look good.

File: deployment/GP02HYPM_Deploy.csv [hunk 8]
```diff
@@ -1,99 +1,99 @@
...
+SR1811,,SKQ201920S,,GP02HYPM-WFP02-05-VEL3DL000,6,1,2018-07-22T21:51:00,2019-09-26T19:10:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-10402,CGINS-VEL3DL-01108,50.06856,-144.79826,,2095,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-00-WFPENG000,6,1,2018-07-22T21:51:00,2019-09-26T22:09:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGCON-GWFPGX-93602,50.06856,-144.79826,,4063,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-01-FLORDL000,6,1,2018-07-22T21:51:00,2019-09-26T22:09:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGINS-FLORDL-02736,50.06856,-144.79826,,4063,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-03-DOSTAL000,6,1,2018-07-22T21:51:00,2019-09-26T22:09:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGINS-DOSTAL-01086,50.06856,-144.79826,,4063,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-04-CTDPFL000,6,1,2018-07-22T21:51:00,2019-09-26T22:09:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGINS-CTDPFL-00107,50.06856,-144.79826,,4063,4222,
+SR1811,,SKQ201920S,,GP02HYPM-WFP03-05-VEL3DL000,6,1,2018-07-22T21:51:00,2019-09-26T22:09:00,CGMGP-02HYPM-00006,CGVEH-GWFPGX-93602,CGINS-VEL3DL-01024,50.06856,-144.79826,,4063,4222,
```
Comment (snwhite) [inline deployment/GP02HYPM_Deploy.csv:85]: Deployment 6: Corrections look good.

File: deployment/GP02HYPM_Deploy.csv [hunk 9]
```diff
@@ -1,99 +1,99 @@
...
+MV1309,,MV1404,,GP02HYPM-WFP03-00-WFPENG000,1,1,2013-07-26T08:11:00,2014-06-14T00:00:00,CGMGP-02HYPM-00001,CGVEH-GWFPGX-77401,CGCON-GWFPGX-77401,50.07067,-144.798,,4061,4219,
+MV1309,,MV1404,,GP02HYPM-WFP03-01-FLORDL000,1,1,2013-07-26T08:11:00,2014-06-14T00:00:00,CGMGP-02HYPM-00001,CGVEH-GWFPGX-77401,CGINS-FLORDL-02350,50.07067,-144.798,,4061,4219,
+MV1309,,MV1404,,GP02HYPM-WFP03-03-DOSTAL000,1,1,2013-07-26T08:11:00,2014-06-14T00:00:00,CGMGP-02HYPM-00001,CGVEH-GWFPGX-77401,CGINS-DOSTAL-01106,50.07067,-144.798,,4061,4219,
+MV1309,,MV1404,,GP02HYPM-WFP03-04-CTDPFL000,1,1,2013-07-26T08:11:00,2014-06-14T00:00:00,CGMGP-02HYPM-00001,CGVEH-GWFPGX-77401,CGINS-CTDPFL-00097,50.07067,-144.798,,4061,4219,
+MV1309,,MV1404,,GP02HYPM-WFP03-05-VEL3DL000,1,1,2013-07-26T08:11:00,2014-06-14T00:00:00,CGMGP-02HYPM-00001,CGVEH-GWFPGX-77401,CGINS-VEL3DL-01025,50.07067,-144.798,,4061,4219,
+#MV1309,,MV1404,,GP02HYPM-MPM01-02-ZPLSGA009,1,1,2013-07-26T08:11:00,2014-06-14T00:00:00,CGMGP-02HYPM-00001,,,50.07067,-144.798,,,4219,Not Deployed
+#MV1309,,MV1404,,GP02HYPM-MPM01-02-ZPLSGA010,1,1,2013-07-26T08:11:00,2014-06-14T00:00:00,CGMGP-02HYPM-00001,,,50.07067,-144.798,,,4219,Not Deployed
```
Comment (johnmlund) [inline deployment/GP02HYPM_Deploy.csv:15]: looks good

File: deployment/GP02HYPM_Deploy.csv [hunk 10]
```diff
@@ -1,99 +1,99 @@
...
+MV1404,,TN323,,GP02HYPM-WFP03-00-WFPENG000,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,CGVEH-GWFPGX-10402,CGCON-GWFPGX-10402,50.07833,-144.80533,,4038,4219,
+MV1404,,TN323,,GP02HYPM-WFP03-01-FLORDL000,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,CGVEH-GWFPGX-10402,CGINS-FLORDL-03283,50.07833,-144.80533,,4038,4219,
+MV1404,,TN323,,GP02HYPM-WFP03-03-DOSTAL000,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,CGVEH-GWFPGX-10402,CGINS-DOSTAL-01480,50.07833,-144.80533,,4038,4219,
+MV1404,,TN323,,GP02HYPM-WFP03-04-CTDPFL000,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,CGVEH-GWFPGX-10402,CGINS-CTDPFL-00134,50.07833,-144.80533,,4038,4219,
+MV1404,,TN323,,GP02HYPM-WFP03-05-VEL3DL000,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,CGVEH-GWFPGX-10402,CGINS-VEL3DL-13551,50.07833,-144.80533,,4038,4219,
+#MV1404,,TN323,,GP02HYPM-MPM01-02-ZPLSGA009,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,,,50.07833,-144.80533,,,4219,Not Deployed
+#MV1404,,TN323,,GP02HYPM-MPM01-02-ZPLSGA010,2,1,2014-06-22T02:14:00,2015-06-02T00:00:00,CGMGP-02HYPM-00002,,,50.07833,-144.80533,,,4219,Not Deployed
```
Comment (johnmlund) [inline deployment/GP02HYPM_Deploy.csv:29]: looks good

File: deployment/GP02HYPM_Deploy.csv (hunk 6 above)
Comment (johnmlund) [inline deployment/GP02HYPM_Deploy.csv:43]: looks good

File: deployment/GP02HYPM_Deploy.csv (hunk 7 above)
Comment (johnmlund) [inline deployment/GP02HYPM_Deploy.csv:57]: looks good

File: deployment/GP02HYPM_Deploy.csv [hunk 11]
```diff
@@ -1,99 +1,99 @@
...
+SR1710,,SR1811,,GP02HYPM-WFP02-05-VEL3DL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-99701,CGINS-VEL3DL-01026,50.1156,-144.9176,,2095,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-00-WFPENG000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGCON-GWFPGX-10403,50.1156,-144.9176,,4063,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-01-FLORDL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGINS-FLORDL-03284,50.1156,-144.9176,,4063,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-03-DOSTAL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGINS-DOSTAL-01478,50.1156,-144.9176,,4063,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-04-CTDPFL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGINS-CTDPFL-00132,50.1156,-144.9176,,4063,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-05-VEL3DL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,50.1156,-144.9176,,4063,4230,
```
Comment (johnmlund) [inline deployment/GP02HYPM_Deploy.csv:71]: looks good

File: deployment/GP02HYPM_Deploy.csv (hunk 8 above)
Comment (johnmlund) [inline deployment/GP02HYPM_Deploy.csv:85]: looks good

File: deployment/GP02HYPM_Deploy.csv [hunk 12]
```diff
@@ -1,99 +1,99 @@
...
+SKQ201920S,,,,GP02HYPM-WFP02-04-CTDPFL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-99701,CGINS-CTDPFL-00115,50.11518,-144.91612,,2095,4237,
+SKQ201920S,,,,GP02HYPM-WFP02-05-VEL3DL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-99701,CGINS-VEL3DL-01026,50.11518,-144.91612,,2095,4237,
+SKQ201920S,,,,GP02HYPM-WFP03-00-WFPENG000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGCON-GWFPGX-10403,50.11518,-144.91612,,4063,4237,
+SKQ201920S,,,,GP02HYPM-WFP03-01-FLORDL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-FLORDL-03284,50.11518,-144.91612,,4063,4237,
+SKQ201920S,,,,GP02HYPM-WFP03-03-DOSTAL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-DOSTAL-01478,50.11518,-144.91612,,4063,4237,
+SKQ201920S,,,,GP02HYPM-WFP03-04-CTDPFL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-CTDPFL-00132,50.11518,-144.91612,,4063,4237,
+SKQ201920S,,,,GP02HYPM-WFP03-05-VEL3DL000,7,1,2019-09-26T00:05:00,,CGMGP-02HYPM-00007,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,50.11518,-144.91612,,4063,4237,
```
Comment (johnmlund) [inline deployment/GP02HYPM_Deploy.csv:99]: looks good

**Pull request merged**